modules/ppcp-button/src/Endpoint/GetOrderEndpoint.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * The endpoint to get a PayPal order.
+ *
+ * @package WooCommerce\PayPalCommerce\Button\Endpoint
+ */
+
+declare(strict_types=1);
+
+namespace WooCommerce\PayPalCommerce\Button\Endpoint;
+
+use Exception;
+use Psr\Log\LoggerInterface;
+use WooCommerce\PayPalCommerce\ApiClient\Endpoint\OrderEndpoint;
+use WooCommerce\PayPalCommerce\ApiClient\Exception\PayPalApiException;
+use WooCommerce\PayPalCommerce\ApiClient\Exception\RuntimeException;
+
+/**
+ * Class GetOrderEndpoint
+ */
+class GetOrderEndpoint implements EndpointInterface {
+
+	const ENDPOINT = 'ppc-get-order';
```

Comment:
Please add visibility.

```suggestion
	const public ENDPOINT = 'ppc-get-order';
```